styx-api-service/src/main/java/com/spotify/styx/StyxApi.java

```diff
@@ -66,6 +66,7 @@
 
   public static final String SCHEDULER_SERVICE_BASE_URL = "styx.scheduler.base-url";
   public static final String DEFAULT_SCHEDULER_SERVICE_BASE_URL = "http://localhost:8080";
+  public static final String COMPONENTS_API_BASE_URL = "styx.components.api.base-url";
```

Comment:
Let's just get rid of this field and leave it empty when posting to API endpoint.